### PR TITLE
Fix Issue 2396 - v2

### PR DESCRIPTION
**Pull request by ya3ya6**
Fix Issue 2396 - v2

### CODE REVIEW

File: __tests__/shared/components/Header/__snapshots__/index.jsx.snap [hunk 1]
```diff
@@ -15,6 +16,39 @@ exports[`Default render 1`] = `
...
+            Object {
+              "href": "https://www.topcoder-dev.com/about",
+              "title": "Overview",
+            },
+            Object {
+              "href": "https://www.topcoder-dev.com/how-it-works/faqs/",
+              "title": "How It Works",
+            },
+            Object {
+              "href": "https://www.topcoder-dev.com/community/learn",
```
Comment (luizrrodrigues) [inline __tests__/shared/components/Header/__snapshots__/index.jsx.snap]: Please use "local" link. 
`/community/learn`

```suggestion
              "href": "/community/learn",
```

File: config/default.js [hunk 2]
```diff
@@ -217,6 +217,39 @@ module.exports = {
...
+          title: 'Overview',
+          href: 'https://www.topcoder-dev.com/about',
+        },
+        {
+          title: 'How It Works',
+          href: 'https://www.topcoder-dev.com/how-it-works/faqs/',
+        },
+        {
+          title: 'Tracks',
+          href: 'https://www.topcoder-dev.com/community/learn',
```
Comment (luizrrodrigues) [inline config/default.js]:
```suggestion
          href: '/community/learn',
```

File: config/production.js [hunk 3]
```diff
@@ -60,6 +60,39 @@ module.exports = {
...
+          title: 'Overview',
+          href: 'https://www.topcoder.com/about',
+        },
+        {
+          title: 'How It Works',
+          href: 'https://www.topcoder.com/how-it-works/faqs/',
+        },
+        {
+          title: 'Tracks',
+          href: 'https://www.topcoder.com/community/learn',
```
Comment (luizrrodrigues) [inline config/production.js]:
```suggestion
          href: '/community/learn',
```

**Comment by ya3ya6** [comment]
@luizrrodrigues  should i use local link for all of pages? even payments? cause most of them doesn't exists in local.

**Comment by luizrrodrigues** [comment]
@ya3ya6 Nope, just to `/community/learn`, others pages not exist.
Just what I comment above. 

I'll apply the patch, 1min,

**Comment by luizrrodrigues** [comment]
@ya3ya6 Looks like I can't push to repo directly, please do changes comment above.
Thanks.

**Comment by ya3ya6** [comment]
@luizrrodrigues Thanks, i'll do it in minutes.

**Comment by ya3ya6** [comment]
@luizrrodrigues Updated.

**Pull request merged**